examples/color_example.cc

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2019 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+//! [complete]
+#include <iostream>
+#include <ignition/math/Color.hh>
+
+int main(int argc, char **argv) 
+{
+
+//! [Create a color]
```

Comment:
Do you plan to add a Color tutorial ? These kind of tags are used there to insert code in the tutorial.

For example [triangle tutorial](https://github.com/ignitionrobotics/ign-math/blob/ign-math6/tutorials/triangle.md) and [online triangle tutorial](https://ignitionrobotics.org/api/math/6.9/angle.html)

Reply:
Oh, I didn't know what they do but sounds like a good idea. Now we have the tutorial. Thanks for the suggestion.